addons/salvage2/salvage2.lua

```diff
@@ -75,9 +77,9 @@ local params = {...};
 				windower.text.set_location('salvage_box2', posx, posy)
 			end
 		elseif params[1]:lower() == "hide" then
-			windower.text.set_visibility('salvage_box2', false)
+			salvage_box2:hide()
```

Comment:
Tabs! They were here before, too, but still, please change all tabs to 4 spaces.